infra/base-images/base-builder/detect_repo.py

```diff
@@ -29,26 +29,37 @@
 
 
 def main():
-  """Function to get a git repos information based on its commit."""
+  """Function to get a git repo's url and name refenced by OSS-Fuzz Dockerfile.
+
+  Raises:
+    ValueError when a commit or a ref is not provided.
+  """
   parser = argparse.ArgumentParser(
-      description='Finds a specific git repo in an oss-fuzz projects docker file.'
+      description="Finds a specific git repo in an oss-fuzz project's docker file."
```

Comment:
single quotes for nondocstrings.

Reply:
I need double quotes for the apostrophe.

Reply:
Same as my reply on max's comment.